src/compositor/seat/touch.c

```diff
@@ -87,7 +87,7 @@ wlc_touch_touch(struct wlc_touch *touch, uint32_t time, enum wlc_touch_type type
          break;
 
          case WLC_TOUCH_MOTION:
-            wl_touch_send_motion(wr, time, slot, wl_fixed_from_double(pos->x), wl_fixed_from_double(pos->y));
+            wl_touch_send_motion(wr, time, slot, pos->x, pos->y);
             break;
```

Comment:
These functions take wl_fixed as argument, hence wl_fixed_from_double is needed.